src/reports/filters.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * @version     CVS: 1.0.0
+ * @package     Joomla.Plugin
+ * @subpackage  Api.reports
+ * @author      Techjoomla <extensions@techjoomla.com>
+ * @copyright   2017 Techjoomla
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+// No direct access.
+defined('_JEXEC') or die;
+
+/**
+ * Tjreports API report class
+ *
+ * @since  1.0.0
+ */
+class ReportsApiResourceFilters extends ApiResource
+{
+	/**
+	 * Function save report.
+	 *
+	 * @return boolean
+	 */
+	public function get()
+	{
+		$user = JFactory::getUser();
```

Comment:
@monikaatvowel  do you want user to login to site first and then use the API?

Reply:
fixed auth code